.gitignore

```diff
@@ -134,3 +133,6 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+#idea
+.idea
```

Comment:
I guess you tried to ignore the idea folder, but it seems to have not been able to ignore it.